esp32_imu_node:
  ros__parameters:
    serial_port: /dev/ttyS1
    baud_rate: 115200
    frame_id: imu_link
    publish_rate: 100.0
    connect_retry_period: 5.0

# JSON format of imu input
# {
  # "accel_x": 0.0, 
  # "accel_y": 0.0, 
  # "accel_z": 9.81, 
  # "gyro_x": 0.0, 
  # "gyro_y": 0.0, 
  # "gyro_z": 0.0, 
  # "quat_w": 1.0, 
  # "quat_x": 0.0, 
  # "quat_y": 0.0, 
  # "quat_z": 0.0
# }